examples/how_to/get_started.py

```diff
@@ -59,9 +59,9 @@
 import spikeinterface.curation as scur
 import spikeinterface.widgets as sw
 
-#  We can also import all submodules at once with this
-#  this internally imports core+extractors+preprocessing+sorters+postprocessing+
-#  qualitymetrics+comparison+widgets+exporters
+# Alternatively, we can import all submodules at once which
+# internally imports core+extractors+preprocessing+sorters+postprocessing+
+# qualitymetrics+comparison+widgets+exporters
```

Comment:
```suggestion
# Alternatively, we can import all submodules at once with `import spikeinterface.full as si` which
# internally imports core+extractors+preprocessing+sorters+postprocessing+
# qualitymetrics+comparison+widgets+exporters. In this case all aliases in the following tutorial
# would be `si`
```

We don't actually mention how to do that here.... If we are going to mention the alternative we might want to show it. I think the `with this` in the older version was meant to be replaced with `import spikeinterface.full as si`. But they never got around to it.